### PR TITLE
[functionapp] fixes #8553 - set correct docker images

### LINKED ISSUE #8553: az create functionapp uses the wrong image for Linux app service function app
**Describe the bug**
"az functionapp create" uses the wrong image for Linux App Service Function apps.
https://github.com/Azure/azure-cli/blob/f1c301b07dc349bbbda30ae83df0809345a88f9e/src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py#L1985

It should instead use one of the following images (based on the runtime)
microsoft/azure-functions-dotnet-core2.0:2.0
microsoft/azure-functions-node8:2.0
microsoft/azure-functions-python3.6:2.0

**To Reproduce**
az functionapp create --resource-group <RG> --os-type Linux --plan <PLAN> --runtime python --name <APPNAME> --storage-account <SA>

**Expected behavior**
This creates a linux appservice python function app, but the image used is not microsoft/azure-functions-python3.6:2.0.

**Environment summary**
Install Method (e.g. pip, interactive script, apt-get, Docker, MSI, edge build) / CLI version (`az --version`) / OS version / Shell Type (e.g. bash, cmd.exe, Bash on Windows)  

**Additional context**

### PR DESCRIPTION
**Pull request by ankitkumarr**
\cc @balag0 @ahmelsayed @panchagnula  @ahmedelnably 

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### CODE REVIEW

File: src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py [hunk 1]
```diff
@@ -1981,7 +1980,9 @@ def create_function(cmd, resource_group_name, name, storage_account, plan=None,
             else:
                 site_config.app_settings.append(NameValuePair(name='WEBSITES_ENABLE_APP_SERVICE_STORAGE',
                                                               value='true'))
-                site_config.linux_fx_version = _format_fx_version('appsvc/azure-functions-runtime')
+                if runtime not in RUNTIME_TO_IMAGE:
```
Comment (yugangw-msft) [inline src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py]: I suggest ensure case insensitive matching: `runtime.lower()`

File: src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_constants.py [hunk 2]
```diff
@@ -12,8 +12,13 @@
...
 OS_DEFAULT = "Windows"
-STATIC_RUNTIME_NAME = "static"  # not an oficial supported runtime but used for CLI logic
+STATIC_RUNTIME_NAME = "static"  # not an official supported runtime but used for CLI logic
 NODE_VERSIONS = ['4.4', '4.5', '6.2', '6.6', '6.9', '6.11', '8.0', '8.1', '8.9', '8.11']
 NETCORE_VERSIONS = ['1.0', '1.1', '2.0']
 DOTNET_VERSIONS = ['3.5', '4.7']
 LINUX_SKU_DEFAULT = "P1V2"
+RUNTIME_TO_IMAGE = {
+    'node': 'mcr.microsoft.com/azure-functions/node:2.0',
+    'dotnet': 'mcr.microsoft.com/azure-functions/dotnet-core:2.0',
```
Comment (ahmelsayed) [inline src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_constants.py]: the image name should be `mcr.microsoft.com/azure-functions/dotnet:2.0`

Comment (ankitkumarr) [reply src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_constants.py]: Oh, right! Thanks @ahmelsayed. I confused this with `microsoft/azure-functions-dotnet-core2.0:2.0`. Fixed now.

File: src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py (hunk 1 above)
Comment (ankitkumarr) [reply src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py]: Thanks @yugangw-msft, updated!

**Comment by ankitkumarr** [comment]
@yugangw-msft this should be ready to merge. I do not have access to merge

**Pull request merged**